help/en/html/apps/DecoderPro/CreateDecoderAdvanced.shtml

```diff
@@ -956,6 +956,8 @@ delay is 500 milliseconds.
 
       <p>You can also use the include mechanism to specify the default value of a variable.</p>
 
+      <p>Note: &lt;defaultItem&gt; must be the first element inside the &lt;variable&gt; element body.</p>
+
```

Comment:
Does this play nice with #11474, or do we now have the same line in two different places?